src/openforms/forms/api/serializers.py

```diff
@@ -392,6 +393,15 @@ class LogicComponentActionSerializer(serializers.Serializer):
             "optional if the action type is `{action_type}`, otherwise required."
         ).format(action_type=LogicActionTypes.disable_next),
     )
+    form_step = serializers.CharField(
```

Comment:
Shouldn't this rather be a `HyperlinkedRelatedField` which takes a URL reference and is resolved to the form step instance?

Mostly reasoning from consistency with other endpoints/serializers where URL references are used throughout the API.

Reply:
I think that with a `HyperlinkedRelatedField` the form steps resolves to the `FormStep` object and then since the action is a `JSONField`  we get an error that it is not serializable. Maybe I should make it a URL field with a validate method that checks that it is an existing formstep?

Reply:
cool, yeah, I figured that would've been the problem